prow/plugins/welcome/welcome.go

```diff
@@ -32,7 +32,7 @@ import (
 
 const (
 	pluginName            = "welcome"
-	defaultWelcomeMessage = "Welcome @{{.AuthorLogin}}! It looks like this is your first PR to {{.Org}}/{{.Repo}} 🎉"
+	defaultWelcomeMessage = "Welcome @{{.AuthorLogin}}! <br> It looks like this is your first PR to <a href=`https://github.com/{{.Org}}/{{.Repo}}`>{{.Org}}/{{.Repo}}</a> 🎉. Please refer to our [pull request process documentation](https://git.k8s.io/community/contributors/guide/pull-requests.md) to help your PR have a smooth ride to approval. \nYou will be prompted by a bot to use commands during the review process. Do not be afraid to follow the prompts! It is okay to experiment. [Here is the bot commands documentation](https://go.k8s.io/bot-commands). \nYou can also check if {{.Org}}/{{.Repo}} has [its own contribution guidelines](https://github.com/{{.Org}}/{{.Repo}}/CONTRIBUTING.md). \nYou may want to refer to our [testing guide](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md) if you run into trouble with your tests not passing. \nIf you are having difficulty getting your pull request seen, please follow the [recommended escalation practices](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#why-is-my-pull-request-not-getting-reviewed). We want to make sure your contribution gets all the attention it needs! <br> Thank you, and welcome to Kubernetes."
```

Comment:
Welcome @guineveresaenger! <br> It looks like this is your first PR to <a href=`https://github.com/kubernetes/test-infra`>kubernetes/test-infra</a> 🎉. Please refer to our [pull request process documentation](https://git.k8s.io/community/contributors/guide/pull-requests.md) to help your PR have a smooth ride to approval. 
You will be prompted by a bot to use commands during the review process. Do not be afraid to follow the prompts! It is okay to experiment. [Here is the bot commands documentation](https://go.k8s.io/bot-commands). 
You can also check if kubernetes/test-infra has [its own contribution guidelines](https://github.com/kubernetes/test-infra/CONTRIBUTING.md). 
You may want to refer to our [testing guide](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md) if you run into trouble with your tests not passing. 
If you are having difficulty getting your pull request seen, please follow the [recommended escalation practices](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#why-is-my-pull-request-not-getting-reviewed). We want to make sure your contribution gets all the attention it needs! <br> Thank you, and welcome to Kubernetes.

Reply:
quick suggestions:
- switch the quote character on the link to `'`
- replace `<br>` with `\n`
- then replace `\n` with `\n\n`
- `CONTRIBUTING.md` should be `/tree/master/CONTRIBUTING.md` - this link doesn't work
- add a trailing smile (`:smiley:` or literal `😃` should work I think)

which looks like:

-----

Welcome @guineveresaenger!

It looks like this is your first PR to <a href='https://github.com/kubernetes/test-infra'>kubernetes/test-infra</a> 🎉. 

Please refer to our [pull request process documentation](https://git.k8s.io/community/contributors/guide/pull-requests.md) to help your PR have a smooth ride to approval.

You will be prompted by a bot to use commands during the review process. Do not be afraid to follow the prompts! It is okay to experiment. [Here is the bot commands documentation](https://go.k8s.io/bot-commands).

You can also check if kubernetes/test-infra has [its own contribution guidelines](https://github.com/kubernetes/test-infra/CONTRIBUTING.md). 
You may want to refer to our [testing guide](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md) if you run into trouble with your tests not passing. 

If you are having difficulty getting your pull request seen, please follow the [recommended escalation practices](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#why-is-my-pull-request-not-getting-reviewed). We want to make sure your contribution gets all the attention it needs!

Thank you, and welcome to Kubernetes. :smiley:

----

thanks again!

Reply:
Looks good overall with the rendering fixes. :+1: 

Thanks a lot for getting this out the door, @guineveresaenger! :blush: :heart: